README.md

```diff
@@ -1 +1,71 @@
-# azure-cosmosdb-mongodbshardedcluster
```

Comment:
what title do you propose? Should I just keep azure-mongodbshardedcluster?

Reply:
This title is irrelevant since I am going to check this code into the azure quickstart templates repo. I'll name the folder as per the guidelines of that repo